anvil-python/anvil/commands/haproxy.py

```diff
@@ -98,6 +112,11 @@ def haproxy_questions() -> dict[str, questions.PromptQuestion]:
             default_value="",
             validation_function=validate_key_file,
         ),
```

Comment:
Since we are going to use the ssl params for both haproxy and maas-region from now on, we should update the entire logic to have a common set ot ssl_questions, displayed in an `ssl` section. In addition, what happens if we have nodes with region role with passthrough but no node with role haproxy? We need to add a prompt to maas-region too. Both of the prompts, whichever asked first should get the answers user provided (if not manifest) and the second one should avoid prompting if someone has already provided the answers.